2.5/Dockerfile.rhel7

```diff
@@ -0,0 +1,66 @@
+FROM rhscl/s2i-base-rhel7:1
+
+# This image provides a Ruby environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV RUBY_MAJOR_VERSION=2 \
+    RUBY_MINOR_VERSION=5 \
+    RUBY_NODEJS_SCL="rh-nodejs6"
+
+ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}" \
+    RUBY_SCL_NAME_VERSION="${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}"
+
+ENV RUBY_SCL="rh-ruby${RUBY_SCL_NAME_VERSION}" \
```

Comment:
You can also merge this ENV with the one below (SUMMARY + DESCRIPTION), as the second one only needs `RUBY_VERSION` defined.

Reply:
OK, I will modify like this.